zendesk_tickets/forms.py

```diff
@@ -0,0 +1,49 @@
+from django import forms
+from django.conf import settings
+from django.template import loader, Context
+from django.utils.translation import ugettext as _
+
+from . import client
+
+
+class BaseTicketForm(forms.Form):
+    referer = forms.CharField(widget=forms.HiddenInput, required=False)
+
+    def _populate_custom_fields(self, context):
+        fields = []
+        for custom_field in settings.ZENDESK_CUSTOM_FIELDS:
+            value = context.get(custom_field)
+            if value is not None:
+                fields.append({
+                    'id': settings.ZENDESK_CUSTOM_FIELDS[custom_field],
+                    'value': value
+                })
+
+    def submit_ticket(self, request, subject, tags,
+                      ticket_template_name, extra_context={}):
+        context = Context(dict(self.cleaned_data, **extra_context))
+        body = loader.get_template(ticket_template_name).render(context)
+
+        client.create_ticket(
+            subject,
+            tags,
+            body,
+            self._populate_custom_fields(context)
+        )
+
+
+class TicketForm(BaseTicketForm):
+    content = forms.CharField(
```

Comment:
This might be clearer as something like `ticket_body` or `ticket_content`, I think?